build.gradle

```diff
@@ -10,6 +10,8 @@ repositories {
 }
 
 dependencies {
+    implementation 'junit:junit:4.13.1'
```

Comment:
Opt: why do you need this?

Reply:
I've been trying something and IDEa has added this dependency automatically, I think. Removed it.